.github/workflows/close-issue-on-release.yml

```diff
@@ -0,0 +1,21 @@
+name: Close issues on release cut
+
+on:
+  release:
+    types: [released]
```

Comment:
Would this still work even if we don't publish releases on GitHub? I am looking at https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=released#release and it seems the event gets triggered on a release being published.

Reply:
Discussed offline. GitHub releases are done along with PyPi releases https://github.com/aws/aws-lambda-builders/releases so this should work as expected.